packages/cozy-intent/src/view/components/WebviewIntentProvider.tsx

```diff
@@ -1,37 +1,81 @@
 import React, { ReactElement, useEffect, useState } from 'react'
+import { Connection, ChildHandshake } from 'post-me'
 
 import { isFlagshipApp } from 'cozy-device-helper'
 
 import { CozyBar } from '../../api/models/applications'
-import { StaticService } from '../../api/services/StaticService'
+import { strings } from '../../api/constants'
+import { TypeguardService } from '../../api/services/TypeguardService'
 import { WebviewConnection } from '../../api/models/environments'
 import { WebviewContext } from '../contexts/WebviewContext'
+import { WebviewMessenger } from '../../api/services/WebviewMessenger'
 import { WebviewService } from '../../api/services/WebviewService'
 
-declare let cozy: CozyBar | undefined
+declare const cozy: CozyBar | undefined
 
 interface Props {
   children?: React.ReactChild
   webviewService?: WebviewService
 }
 
+const getBarInitAPI = ():
+  | ((webviewContext: WebviewService) => void)
+  | undefined => {
+  try {
+    return cozy?.bar?.setWebviewContext
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.error((err as Error).stack)
+
+    // If we fail to get the cozy-bar, we want to assume it doesn't exist
+    return undefined
```

Comment:
return null then

Reply:
Undefined is more suited to describe the eventual runtime type of a failed cozy-bar fetch